src/qibolab/_core/instruments/qblox/config.py

```diff
@@ -102,8 +106,15 @@ def module(
     for probe, lo in los:
         if lo is None:
             continue
-        n = mod_channels[probe].ports[0]  # TODO: check it is the correct path
-        getattr(mod, f"out{n}_lo_freq")(cast(OscillatorConfig, configs[lo]).frequency)
+        n = mod_channels[probe].ports[0] - 1
+
+        attr = f"out{n}_in{n}_lo_freq" if mod.is_qrm_type else f"out{n}_lo_freq"
+        getattr(mod, attr)(cast(OscillatorConfig, configs[lo]).frequency)
+
+
+def radians_to_degrees(angle):
+    """Convert iq-angle to degrees in (0, 360)."""
+    return (angle % (2 * math.pi)) * 180 / math.pi
```

Comment:
Nothing bad, but just to save some characters (and maintenance), you may use [`np.degrees`](https://numpy.org/doc/stable/reference/generated/numpy.degrees.html).

Reply:
Thanks, fixed in 47af3936c7d02eb579ec0366804088a7cd7b1b33.